vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System.ComponentModel.Composition
+open System.Runtime.InteropServices
+
+open Microsoft.VisualStudio.FSharp.UIResources
+open SettingsPersistence
+open OptionsUIHelpers
+
+//autoproperties can be used to conveniently define defaults and faciliate data binding,
+//but the properties shouldn't be mutated outside of the Options dialog UI
+type IntelliSenseOptions() =
```

Comment:
Because the settings "framework" treats settings types as immutables. You can use a record or anything there if you want to write your own databinding and view models etc.